frontend/express/views/dashboard.html

```diff
@@ -1952,74 +1940,88 @@ <h4><a href="#/analytics/events/key/{{encodeURIComponent this.name}}">{{this.nam
                 return view + parts.join("/");
             };
 
-            //track pageviews automatically
-            Countly.track_pageview(Countly.getViewName());
+            if (countlyGlobal.countly_tracking) {
+                //initializing countly with params
+                Countly.init({
+                    app_key: "<%= frontend_app || "e70ec21cbe19e799472dfaee0adb9223516d238f" %>",
+                    url: "<%= frontend_server || "https://stats.count.ly/" %>",
+                    device_id: window.location.hostname,
```

Comment:
We should not use location hostname, because it can be different (multiple domain use, etc)
We should use api.domain setting
and if we detect that it has changed, we should do user merging

Reply:
How do we detect if the domain has changed?
And how do we do user merging?